src/css/components/_layout.scss

```diff
@@ -37,6 +43,28 @@
   box-sizing: inherit;
 }
 
+/* Fill the width of the containing element and use padding to create the
+   desired aspect ratio. Default to 16x9 unless another ratio is given. */
+.video-js.vjs-fluid,
+.video-js.vjs-16-9 {
+  width: 100%;
+  max-width:100%;
+  height: 0;
+  padding-top: 56.25%;
```

Comment:
I don't think it's necessary, but it might actually make this more readable to refactor this into a mixin.

```scss
@mixin generate-player-sizing($ratio) {
  width: 100%;
  max-width: 100%;
  height: 0;
  padding-top: 100% * $ratio;
}

.video-js.vjs-fluid,
.video-js.vjs-16-9 {
  @include generate-player-sizing(9/16);
}

.video-js.vjs-4-3 {
  @include generate-player-sizing(3/4);
}
```

That mixin name sucks, but you get the picture.